canonical-web-and-design/ubuntu.com
# Add responsive cancel modal

## Done

- Add cancel component.
- Make cancel component responsive.

## QA

- Visit /advantage?test_backend=true.
- Click "You can cancel this subscription online or contact us."
- You should see a modal.
- Resize your browser down to mobile size.
- The cancel button should stay at the bottom of the modal.


## Issue / Card

Fixes: canonical-web-and-design/commercial-squad#115.

## Screenshots

<img width="743" alt="Screen Shot 2021-08-04 at 4 36 42 pm" src="https://user-images.githubusercontent.com/361637/128133495-a01f6321-4853-423b-82ca-2f46cd96736c.png">

<img width="344" alt="Screen Shot 2021-08-04 at 4 38 18 pm" src="https://user-images.githubusercontent.com/361637/128133649-b15d0a5e-9da4-46eb-b3a5-28af3c110037.png">
